### ISSUE TITLE
import this

### ISSUE THREAD

**Issue by jackiekazil** [oboechick/pyfanfic#4]
import this feels like it should be in the this, but I don't feel right submitting it.

**Comment by thursdayb** [comment]
Maybe fan art about import this?

**Comment by jackiekazil** [comment]
But isn't `import this` art in itself?  🍿

**Comment by thursdayb** [comment]
Absolutely! Art inspires art!

**Comment by jackiekazil** [comment]
Would you say fan art about `import this` then is a derivative work?

**Comment by thursdayb** [comment]
yes and potentially a delightfully derivative work

**Comment by jackiekazil** [comment]
Potentially is the key word.

**Comment by jackiekazil** [comment]
I would like to submit this ticket as art. Do I need to file another ticket to submit this ticket?

**Comment by jackiekazil** [comment]
![giphy 1](https://user-images.githubusercontent.com/166734/40264520-b9d130ac-5af3-11e8-8430-8bb9129a1055.gif)

**Comment by thursdayb** [comment]
Nah, just figure out how you want it printed. Write a description and put it in the art folder. 

This project is going to get very meta.
